### ISSUE TITLE
Missing separate path check in variant configuration file

### ISSUE THREAD

**Issue by HolQue** [test-fullautomation/robotframework-testsuitesmanagement#106]
A test runs with variant1 selected in command line, based on:

```
"variant1": {
             "name": "exercise-pg_config_variant1.json",
             "path": ".../config/"
            },
```

In case of I make the name invalid
`"name": "invalid.json",`
a proper error message is given (file not found).

But if I make the path invalid instead
`"path": ".../invalid/"`
only an `Unknown exception` is given, but no concrete error message

I would prefer to have an error message telling that the path does not exist.

**Comment by namsonx** [comment]
Hello Holger,

I created the pull-request #120 for this ticket. The log now shows like below:

.. code:: 

    ==============================================================================
    Exercise-06
    ==============================================================================
    [ ERROR ] Testsuite management - Loading configuration level 2 failed! 
              Could not find out config directory: '.../invalid/' which is set in 'C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\robotframework-tutorial\900_Building_testsuites\exercise-06\config\exercise-06_variants.json'
    
    Test Case exercise-06 :: exercise-06                                  | UNKNOWN |
    Parent suite setup unknown:
    Unknown exception occurs. Details: Loading configuration level 2 failed!
    ------------------------------------------------------------------------------
    Exercise-06                                                           | UNKNOWN |
    Suite setup unknown:
    Unknown exception occurs. Details: Loading configuration level 2 failed!
    
    1 test, 0 passed, 0 failed, 1 unknown
    ==============================================================================
    Debug:   C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\debug_log.log
    Output:  C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\output.xml
    Log:     C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\log.html
    Report:  C:\Users\mas2hc\Desktop\data_storage\workspace\ROBFWAIO-OSS\report.html

Thank you,
Son

**Comment by test-fullautomation** [comment]
solved and already merged with #120

**Comment by HolQue** [comment]
Works fine now!

**Comment by test-fullautomation** [comment]
solved and already merged with #120
